Modules/Daq/include/Daq/DaqTask.h

```diff
@@ -52,12 +49,32 @@ class DaqTask final : public TaskInterface
   void reset() override;
 
  private:
-  TH1F* mPayloadSize;
-  TGraph* mIds;
-  int mNPoints;
-  TH1F* mNumberSubblocks;
-  TH1F* mSubPayloadSize;
-  //  UInt_t mTimeLastRecord;
+  void printInputPayload(const header::DataHeader* header, const char* payload);
+  void monitorInputRecord(o2::framework::InputRecord& inputRecord);
+  void monitorRDHs(o2::framework::InputRecord& inputRecord);
+
+  // ** general information
+
+  std::map<o2::header::DAQID::ID, std::string> mSystems;
+
+  // ** objects we publish **
+
+  // Message related
+  // Block = the whole InputRecord, i.e. the thing we receive and analyse in monitorData(...)
+  // SubBlock = a single input of the InputRecord
+  TH1F* mInputRecordPayloadSize; // filled with the sum of the payload size of all the inputs of an inputrecord
+  TH1F* mNumberInputs;           // filled with the number of inputs in each InputRecord we encounter
+  TH1F* mInputSize;              // filled with the number of inputs in each InputRecord we encounter
+  TH1F* mNumberRDHs;             // filled with the number of RDHs found in each InputRecord we encounter
```

Comment:
You might want to use ` = nullptr`, instead of setting the pointers in constructors. It is said to be faster to compile and I think it is clearer & easier to maintain.